### ISSUE TITLE
Can't build Songbird 0.4.3

### ISSUE THREAD

**Issue by ThalusA** [serenity-rs/songbird#248]
**Songbird version:** 0.4.3

**Rust version (`rustc -V`):** 1.74.1

**Serenity/Twilight version:** 1.22.2

**Description:**
I'm getting this error when building:
```
error[E0034]: multiple applicable items in scope
   --> /home/thalusa/.cargo/registry/src/index.crates.io-6f17d22bba15001f/songbird-0.4.3/src/input/adapters/async_adapter.rs:159:47
    |
159 |         let (bytes_in, bytes_out) = SharedRb::new(buf_len).split();
    |                                               ^^^ multiple `new` found
    |
    = note: candidate #1 is defined in an impl for the type `ringbuf::SharedRb<Owning<[MaybeUninit<T>; N]>>`
    = note: candidate #2 is defined in an impl for the type `ringbuf::SharedRb<Heap<T>>`
```

**Steps to reproduce:**
Build using this Cargo.toml file:
```toml
[dependencies]
rodio = "=0.19.0"
songbird = "=0.4.3"
serenity = { version = "=0.12.2", features = [ "voice" ]}
tokio = { version = "=1.39.3", features = ["full"] }
chrono = "=0.4.38"
dotenv = "=0.15.0"
byteorder = "=1.5.0"
futures = "0.3.30"
```

**Comment by jamesbt365** [comment]
agerasev/ringbuf#37

**Comment by FelixMcFelix** [comment]
Closed by ec665a8.

**Comment by ThalusA** [comment]
`ringbuf = "=0.4.1"` in  Cargo.toml seems to fix this problem
